lib/internal/fs/cp/cp.js

```diff
@@ -306,7 +306,14 @@ function onDir(srcStat, destStat, src, dest, opts) {
 }
 
 async function mkDirAndCopy(srcMode, src, dest, opts) {
-  await mkdir(dest);
+  try {
+    await mkdir(dest);
+  } catch (error) {
+    // If the folder already exists, skip it.
+    if (error.code !== 'EEXIST')
```

Comment:
This pull-request needs a test.

Reply:
I added a test and adjusted it to ignore this error only when the `errorOnExist` parameter is false